ocpp-common/src/test/java/eu/chargetime/ocpp/test/SimpleRequestDispatcherTest.java

```diff
@@ -65,6 +65,6 @@ public void fulfill_throwsException_completesWithException() throws UnsupportedF
     sut.fulfill(promise, eventsMock, null);
 
     // Then
-    assertThat(result[0], is(expectedException));
+    assertEquals(result[0], expectedException);
```

Comment:
This does not seem to be a Mockito issue. I'm not saying it's wrong, I'm just curious to know why this was changed? Do you prefer one over the other, and if so why?

Reply:
Yes just a small preference.